test/mechanics/timeIntegration/RK4.cpp

```diff
@@ -0,0 +1,57 @@
+#include "BoostUnitTest.h"
+
+#include <iostream>
+#include <Eigen/Core>
```

Comment:
Travis error here. Use `<eigen3/Eigen/Core>` for now. This is IMO an issue in our cmakelists.  @Psirus ??? :)